multi_physics/QED/python_bindings/pxr_qed.cpp

```diff
@@ -0,0 +1,1352 @@
+/**
+* This file contains the python bindings for the PICSAR QED library
+*/
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+
+#ifdef PXQEDPY_HAS_OPENMP
+    #include <omp.h>
+#endif
```

Comment:
I am afraid that moving `pybind11/pybind11.h` might cause some issues. Indeed, `pybind11/pybind11.h` includes in turn `Python.h`, which apparently defines macros affecting standard library headers on some systems (https://docs.python.org/3/c-api/intro.html#include-files). This link recommends to include `Python.h` before any standard library header. Since the QED library headers include some standard library headers, I opted to put `pybind11.h` at the top of the inclusion list...